jobs/pipelines/powervm/custom-powervm-jobs/custom-ocp4.x-powervm-p8-deploy/Jenkinsfile

```diff
@@ -0,0 +1,284 @@
+                    sandbox: false,
+                    script:
+                        'return["min:selected","max"]'
+                ]
+            ]
+        ]
+    ])
+])
+pipeline {
+    agent {
+        docker {
+            image 'quay.io/powercloud/inbound-agent:4.6-1.2'
+            args '-v /etc/resolv.conf:/etc/resolv.conf'
+            label 'jump-vpc-x86_64'
+        }
+    }
+    parameters {
+        string(defaultValue: '', description: 'Build(quay image or build number)', name: 'Build')
+        string(defaultValue: '0', description: 'Enter time(in Minutes) to retain the cluster', name: 'KeepFor')
+        booleanParam(defaultValue: false, name: 'Rune2e', description: 'Select if you want to run e2e tests. E2e runs after cluster upgrade')
+    }
+
+    environment {
+        //users and credentials. All must be defined in Jenkins Credentials
+        GITHUB_USER = credentials('GITHUB_USER')
+        GITHUB_TOKEN = credentials('GITHUB_TOKEN')
+        OS_USERNAME = credentials('OS_USERNAME')
+        OS_PASSWORD = credentials('OS_PASSWORD')
+        DOCKER_USER = credentials('DOCKER_USER')
+        ARTIFACTORY_USER = credentials('ARTIFACTORY_USER')
+        ARTIFACTORY_TOKEN = credentials('ARTIFACTORY_TOKEN')
+        REDHAT_USERNAME = credentials('REDHAT_USERNAME')
+        REDHAT_PASSWORD = credentials('REDHAT_PASSWORD')
+        PULL_SECRET = credentials('PULL_SECRET')
+
+        //Parameters
+        OCP_RELEASE="${params.Release}"
+        CONFIG_TYPE="${params.Config}"
+        BUILD="${params.Build}"
+        TIMEOUT = "${params.KeepFor}"
+        ENABLE_E2E_TEST="${params.Rune2e}"
+
+        //Env constants
+        HARDWARE_CHOSE = "P8"
+        AVAILABILITY_ZONE = "zstream"
+        TARGET = "deploy-openshift4-powervc"
+        TEMPLATE_FILE = ".${TARGET}.tfvars.template"
+        POWERVS = false
+        SCRIPT_DEPLOYMENT = false
+        WAIT_FOR_DEBUG = "0"
+        REDHAT_RELEASE = "8.3"
+        ENABLE_SCALE_TEST="false"
+        GOLANG_TARBALL="https://dl.google.com/go/go1.16.5.linux-ppc64le.tar.gz"
+    }
+    stages {
+        stage('Clone ansible extra') {
+            when { expression { return env.ENABLE_E2E_TEST == "true"; } }
+            steps {
+                cloneRepo("git@github.ibm.com:redstack-power/ocp4_ansible_extras.git", "ansible_extra")
+            }
+        }
+        stage('Setup Common Environment Variables') {
+            steps {
+                setupCommonEnvironmentVariables()
+                setupClusterConfig("${CONFIG_TYPE}")
+                script {
+                    wrap([$class: 'BuildUser']) {
+                        env.INSTANCE_NAME = "${BUILD_USER_FIRST_NAME}"
+                        env.INSTANCE_NAME = env.INSTANCE_NAME.toLowerCase()
+                    }
+                    env.TERRAFORM_VER = "0.13.0"
+                    env.OS_TENANT_NAME="icp-test"
+                    env.AUTH_URL="https://scnlpowercloud.pok.stglabs.ibm.com:5000/v3/"
```

Comment:
Change all the instances of env.AUTH_URL to env.OS_AUTH_URL to match with openstack env variable